cmd/bundle/dependencies.go

```diff
@@ -0,0 +1,78 @@
+package bundle
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/databricks/cli/bundle/deploy/terraform"
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/flags"
+	"github.com/spf13/cobra"
+)
+
+type Dependencies struct {
+	Terraform *terraform.TerraformMetadata `json:"terraform"`
+}
+
+func newDependenciesCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "internal-dependencies",
```

Comment:
To stick closer to CLI naming conventions/guidelines, could this maybe be a shorter one word verb? Like `list`? Or possibly this could be a command that might do more than just list internal dependencies in the future, so the verb command with an internal dependencies flag? Or another thought - `bundle namespace`? Or add to the bundle summary command a flag for the namespace? (Random ideas!)